pkg/storage/db.go

```diff
@@ -864,11 +864,13 @@ func (db *DB) FetchByARNID(ctx context.Context, when time.Time, arnID string) ([
 			hasTag = true
 		}
 
-		tempChampionsMap[chLogin+chEmail] = domain.Person{
-			Login: &chLogin,
-			Email: &chEmail,
-			Name:  &chName,
-			Valid: &chValid,
+		if chLogin != nil {
+			tempChampionsMap[*chLogin] = domain.Person{
```

Comment:
I'm not 100% sure about this change. Apparently the login is unique, and I assume the same champion would not appear for the same resource twice. But please correct me if I'm wrong.

Reply:
It's technically possible an account owner could also be an account champion although I don't know of any cases in our data where this is true.

Reply:
In this case the old way this was done would still cause issues correct?

Reply:
Cause what specific issues?

Reply:
I am looking at how the IP based queries are executed and the logic is slightly different which is interesting, is it possible the issue exists there as well?

Reply:
I guess I just want to know why we had the map key as chLogin+chEmail before? Is there some risk to just using the login?

Reply:
Sorry, to clarify there was no issue that I know of in the previous code. I can add a check for nil on both the login and email, I just don't know if that is necessary/preferred.

Reply:
Login is guaranteed to be unique so I am unsure why they would combine the two for a key also